client/reader/discover/post-attribution.jsx

```diff
@@ -61,7 +61,13 @@ class DiscoverPostAttribution extends React.Component {
 					? <img className="gravatar" src={ encodeURI( attribution.avatar_url ) } alt="Avatar" width="20" height="20" />
 					: <Gridicon icon="arrow-right" /> }
 				<span className="discover-attribution__text">
-					{ translate( 'Originally posted by' ) }&nbsp;
+          { translate( 'Post', { context: 'random context' } ) }&nbsp;
+
+          { translate( 'Some string', { context: 'This context is very long. Really long' } ) }
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?